tests/basetest.rb

```diff
@@ -133,7 +133,6 @@ def setup
   def teardown
     @device.close unless @device.nil?
     @device = nil
-    GC.start
```

Comment:
Doesn't look like it impacts memory usage, as recorded with `/usr/bin/time --format='(%M kb max)' rake test`.

without GC line: `(77996 kB max)`
with GC line added back in: `(77516 kB max)`